.circleci/generate_circleci_config.py

```diff
@@ -11,6 +11,7 @@
     "1.10.5": ["alpine3.10", "buster", "rhel7"],
     "1.10.6": ["alpine3.10", "buster"],
     "1.10.7": ["alpine3.10", "buster"],
+    "1.10.10.dev": ["alpine3.10", "buster"],
```

Comment:
Make this an OrderedDict so that the rendered template has less diff please.

Reply:
Good point, will do

Reply:
Done

Reply:
It is in correct order in this PR, maybe previous PR had a different order